### DISCUSSION TITLE
[Feature] Goto Photo from Memories

### DISCUSSION THREAD

**Discussion by elit3ge** [immich-app/immich#3845] [Feature Request]
### The feature

Would it be possible to have a button to go to the photo that is appearing in memories? I’d like to be able to have a look at the metadata of the photo or delete the photo. At the moment I have to scroll through the years to get to say 8 years ago and find a specific photo. 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by daniele-athome** [comment]
Probably blocked by #2906.

**Comment by dabenzel** [comment]
I'd also like to jump to the Photo in the Timeline from the memories.

**Comment by bo0tzz** [comment]
This is now supported
